src/utils.jl

```diff
@@ -42,3 +42,14 @@ Unlike [`sample!`](@ref), does not produce side effects.
 sample(a::UnitRange, k::Integer; exclude = ()) = sample!(getRNG(), collect(a), k; exclude = exclude)
 
 getRNG(seed::Integer = -1) = seed >= 0 ? MersenneTwister(seed) : Base.Random.GLOBAL_RNG
+
+"""
+    insorted(x, val)
+
```

Comment:
`in` has `in(item, collection)`, contrary to LightGraphs convention `has_edge(graph, edge)` and `searchsorted(collection, item)`. I think this is fine.

Reply:
They did that so that infixing would work, I think.

Reply:
On second thought I think we should change `insorted` to be `(item, collection)`. I'll make this change.